### PR TITLE
FRONT-1573: "My delegation" incorrectly shows zero

### PR DESCRIPTION
**Pull request by mondoreale**
Looks like this should do it. The idea is a bit different.

For amounts, instead of using a sum of delegator's amounts we use a sum of their operator token balance rated at `Operator#exchangeRate`. Quite a change.

For percentages/fractions, we divide the above by operator token's total supply, also exchange-rated.

@pmichalowski let's coop on this. Seems to me you're working on something alike.

### CODE REVIEW

File: src/getters/index.ts [hunk 1]
```diff
@@ -749,21 +749,22 @@ export function getSpotApy<
...
 ): BN {
     const addr = address.toLowerCase()
 
-    return delegators.reduce(
-        (sum, { delegator, amount }) =>
-            delegator.toLowerCase() === addr ? sum.plus(amount) : sum,
-        toBN(0),
-    )
+    return delegators
+        .reduce(
```
Comment (mondoreale) [inline src/getters/index.ts:760]: This I'm not sure of. Waiting for @jtakalai to confirm (on Slack) if a single delegator can appear twice in the collection.

- If no, they we can `find` the one (skip the reduce).
- If yes, then what do we sum? `operatorTokenBalanceWei` sounds like it's already all they got.

File: src/getters/index.ts [hunk 2]
```diff
@@ -782,13 +783,15 @@ export function getDelegationFractionForWallet(
     operator: ParsedOperator,
     { offset = toBN(0) }: { offset?: BN } = {},
 ) {
-    const value = operator.valueWithoutEarnings.plus(offset)
+    const total = operator.operatorTokenTotalSupplyWei
+        .plus(offset)
+        .multipliedBy(operator.exchangeRate)
```
Comment (mondoreale) [inline src/getters/index.ts:788]: Question here: do we rate it before or after applying the offset. I.e. this or
```ts
.multipliedBy(operator.exchangeRate)
.plus(offset)
```

Comment (pmichalowski) [reply src/getters/index.ts:788]: I think we should be adding the offset after multiplying by exchange rate as the offset is in DATA

Comment (mondoreale) [reply src/getters/index.ts:788]: Thought so. I'll update!

**Comment by mondoreale** [comment]
This should do it.

**Pull request merged**